### PR TITLE
RangeUtils: migrate logic from kbn to grafana/data

### PR DESCRIPTION
**Pull request by ryantxu**
Pulling this out of #21835 

`kbn.ts` still holds lots of interesting interval math logic... that makes it hard to use outside core ;(

### CODE REVIEW

File: public/app/features/variables/interval/actions.ts [hunk 1]
```diff
@@ -29,22 +28,22 @@ export const updateIntervalVariableOptions = (
 };
 
 export interface UpdateAutoValueDependencies {
-  kbn: typeof kbn;
+  rangeUtil: any; // typeof rangeUtil?
```
Comment (ryantxu) [inline public/app/features/variables/interval/actions.ts:31]: @hugohaggmark any ideas here?  `rangeUtil` is an import namespace, not a real object like kbn

Comment (hugohaggmark) [inline public/app/features/variables/interval/actions.ts:31]: What about this?
```suggestion
  calculateInterval: typeof calculateInterval;
```

Comment (hugohaggmark) [reply public/app/features/variables/interval/actions.ts:31]: I think I only use it as a dependency to simplify testing :)

Comment (hugohaggmark) [reply public/app/features/variables/interval/actions.ts:31]: So you'll need to update some tests as well, but should be straightforward. Let me know if I can help out

Comment (ryantxu) [reply public/app/features/variables/interval/actions.ts]: that worked!  thank you

File: public/app/core/utils/kbn.ts [hunk 2]
```diff
@@ -25,123 +25,16 @@ const kbn = {
...
-    if (numSeconds) {
-      return numSeconds + 's';
-    }
-    const numMilliseconds = Math.floor(seconds * 1000.0);
-    if (numMilliseconds) {
-      return numMilliseconds + 'ms';
-    }
-
-    return 'less than a millisecond'; //'just now' //or other string you like;
+  /** deprecated since 7.2, use grafana/data */
```
Comment (dprokop) [inline public/app/core/utils/kbn.ts:34]: I think adding `@depreceted` annotation will make the code editors highlight the method as deprecated. In Webstorm it looks like this:
image

**Review by dprokop — APPROVED** [review]
This is gold! We need to add this to the changelog though :)

**Pull request merged**